pkcs11/testdata/ec.cert.pem

```diff
@@ -1,10 +1,10 @@
 -----BEGIN CERTIFICATE-----
-MIIBTjCB9gIJAJ5bWBxZlNa/MAoGCCqGSM49BAMCMDAxCzAJBgNVBAYTAlVTMQsw
-CQYDVQQIDAJDQTEUMBIGA1UEAwwLZXhhbXBsZS5jb20wHhcNMjEwMzI2MTczMzEy
-WhcNMjEwNDI1MTczMzEyWjAwMQswCQYDVQQGEwJVUzELMAkGA1UECAwCQ0ExFDAS
-BgNVBAMMC2V4YW1wbGUuY29tMFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAE2eGr
-vNUo5MgKgeimjt3t1ZkM04DN4r7xguptdaR/9erKP0L9nODSr3fWfeKDRx/FClt/
-nZIUwV8q6JbzbuVaDjAKBggqhkjOPQQDAgNHADBEAiBSy79qkY6o1R0uZpjSTVWM
-yJiXIpOml5o+qqyGTr5lcwIgJK8CPd0rmdiUuD5G2QrK1+cZ2ZvvPwQ3Z2l/QAFk
-IjM=
+MIIBTjCB9gIJAOBDOh1JA/40MAoGCCqGSM49BAMCMDAxCzAJBgNVBAYTAlVTMQsw
```

Comment:
Can you please add the steps taken for generating these certs. We eventually would be getting rid of these and instead would generate certs on the fly for unit-tests which should reduce these issues in the future.